From 999c1d6bc999d67657033ec780a561aa172c75a9 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Fri, 11 Nov 2011 10:12:24 -0800
Subject: [PATCH 0262/3390] Linux 3.0.9

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9f6e3cd..438c11a 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 0
-SUBLEVEL = 8
+SUBLEVEL = 9
 EXTRAVERSION =
 NAME = Sneaky Weasel
 
-- 
2.5.0

