From 7a576d2dcd5807a7310032b380442eeba8c1c293 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Sat, 26 Nov 2011 09:11:26 -0800
Subject: [PATCH 0343/3390] Linux 3.0.11

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 36036d1..eaa96c5 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 0
-SUBLEVEL = 10
+SUBLEVEL = 11
 EXTRAVERSION =
 NAME = Sneaky Weasel
 
-- 
2.5.0

