From d986a8dbfd7358bfbda116650c4caf8a3b90d865 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Fri, 9 Dec 2011 08:53:50 -0800
Subject: [PATCH 0426/3390] Linux 3.0.13

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 993fe05..5ccc962 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 0
-SUBLEVEL = 12
+SUBLEVEL = 13
 EXTRAVERSION =
 NAME = Sneaky Weasel
 
-- 
2.5.0

