From c6d7c4dbff353eac7919342ae6b3299a378160a6 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 3 Jan 2012 10:50:13 -0800
Subject: [PATCH 0475/3390] Linux 3.0.15

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f4f577b..5b8c185 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 0
-SUBLEVEL = 14
+SUBLEVEL = 15
 EXTRAVERSION =
 NAME = Sneaky Weasel
 
-- 
2.5.0

